src/inc/regdisp.h

```diff
@@ -8,7 +8,7 @@
 
 #ifdef DEBUG_REGDISPLAY
 class Thread;
-
+struct REGDISPLAY;
```

Comment:
It doesn't have to be declared here if we move the function prototype (wrapped in the same ifdef) to line 340, that means after REGDISPLAY is defined for all the targets.